packages/action/src/utils.ts

```diff
@@ -208,15 +193,15 @@ export const createReleases = curry(
           console.log(
             `uploading asset ${asset.name} for ${pipe.pkg}@${pipe.pkgFile.version}`
           );
-          const uploadedAsset = await octokit.repos
+          const uploadedAsset = await octokit.rest.repos
             .uploadReleaseAsset({
               owner,
               repo,
               release_id: releaseResponse.id,
               name: asset.name,
-              data: fs.readFileSync(asset.path),
+              data: fs.readFileSync(asset.path, { encoding: "utf-8" }),
```

Comment:
We can't use a string here, we need to keep the old code. See https://github.com/tauri-apps/tauri-action/pull/45

Reply:
Oooh, good catch. I bet that was the poor types pushing it towards that.

Reply:
Hmm, even with their types directly it asks for a string.